Tasks/Shurko/Task2/Task2.py

```diff
@@ -0,0 +1,114 @@
+from datetime import datetime
+
+minut_word = {0: 'ноль', 1 : 'одна', 2 : 'две', 3 : 'три', 4 : 'четыре', 5 : 'пять',    6 : 'шесть', 7 : 'семь', 8 : 'восемь', 9 : 'девять',
+                10 : 'десять', 20 : 'двадцать', 30 : 'тридцать', 40 : 'сорок',   50 : 'пятьдесят',
+                11 : 'одиннадцать', 12 : 'двенадцать', 13 : 'тринадцать',    14 : 'четырнадцать', 15 : 'пятнадцать', 16 : 'шестнадцать',    17 : 'семнадцать', 18 : 'восемнадцать', 19 : 'девятнадцать'}
+
+half_hour = {0: 'первого', 1: 'второго', 2: 'третьего', 3: 'четвертого', 4: 'пятого', 5: 'шестого', 6: 'седьмого', 7: 'восьмого', 8: 'девятого', 9: 'десятого',  10: 'одинадцатого', 11: 'двенадцатого'}
+
+
+
+def minutes_in_words (my_time):
+    
+    my_time = int(my_time)
+
+    def minut_ends(var):
+        s = minut_word.get(var)
+        s = s[len(s)-1]
+        if s == "е" or s == "и":
+            return(" минуты")
+        elif s== "а":
+            return(" минута")
+        else:
+            return(" минут")
```

Comment:
the main question here - why do you need this function defined inside the **minutes_in_words**? by doing so you restrict the scope of it, and I have doubts that this is a deliberate choice; it would be better to define the function on the global level and just call it here, for the first time I will consider it as a convention break, the next time it will be a structural mistake, so -5 points for now